Translation/Updater.php

```diff
@@ -277,5 +284,23 @@ private function setConfig(Config $config)
                 }
             }
         }
+
+        //keep old translations translated
+        if ($this->config->isKeepOldTranslationsMessages()) {
+
+            $locale = $this->scannedCatalogue->getLocale();
+            /** @var MessageCatalogue $domainCatalogue */
+            foreach ($this->scannedCatalogue->getDomains() as $domainCatalogue) {
+
+                /** @var Message $message */
+                foreach ($domainCatalogue->all() as $message) {
+
+                    $translated = $this->translator->trans($message->getId(), array(), $message->getDomain(), $locale);
+                    $message->setLocaleString($translated);
```

Comment:
So this seems odd to me that you are translating as you go. If all you want to do is keep any messages that were already there, why is this necessary?